pwndbg/commands/ipython_interactive.py

```diff
@@ -0,0 +1,42 @@
+"""
+Command to start an interactive IPython prompt.
+"""
+import sys
+from contextlib import contextmanager
+
+import gdb
+
+import pwndbg.commands
+
+
+@contextmanager
+def switch_to_ipython_env():
+    """We need to change stdout/stderr to the default ones, otherwise we can't use tab or autocomplete"""
+    # Save GDB's stdout and stderr
+    saved_stdout = sys.stdout
+    saved_stderr = sys.stderr
+    # Use Python's default stdout and stderr
+    sys.stdout = sys.__stdout__
+    sys.stderr = sys.__stderr__
+    yield
+    # Restore GDB's stdout and stderr
+    sys.stdout = saved_stdout
+    sys.stderr = saved_stderr
+    # Restore Python's default ps1 and ps2 for GDB's `pi` command
+    sys.ps1 = ">>> "
+    sys.ps2 = "... "
```

Comment:
Are we sure those are the only things we need to restore? Can you double check `sys.__dict__` before and after?

Reply:
Oh, I also found that `sys.excepthook` changed.
I'll restore it, too.